packages/web/src/common/store/lineup/sagas.ts

```diff
@@ -72,10 +72,12 @@ function* filterDeletes<T extends Track | Collection>(
   const getFeatureEnabled = yield* getContext('getFeatureEnabled')
   yield* call(remoteConfig.waitForRemoteConfig)
 
-  const isUSDCGatedContentEnabled = getFeatureEnabled(
```

Comment:
This was returning a promise which was always truthy when checked below.

This change causes the flag to be read as `false` when it's set to disabled.